internal/.repo-metadata-full.json

```diff
@@ -838,5 +838,13 @@
     "client_library_type": "generated",
     "docs_url": "https://pkg.go.dev/cloud.google.com/go/webrisk/apiv1beta1",
     "release_level": "beta"
+  },
+  "cloud.google.com/go/workflows/apiv1beta": {
+    "distribution_name": "cloud.google.com/go/workflows/apiv1beta",
```

Comment:
apiv1beta, not apiv1beta**1**? Generation works, so this seems correct, but wanted to confirm since this flows into package name.

Reply:
Yes, `apiv1beta`. Historically if the pathing in googleapis does not contain a number suffix neither does our package here.